clang-tools-extra/clang-tidy/llvmlibc/InlineFunctionDeclCheck.cpp

```diff
@@ -79,6 +79,10 @@ void InlineFunctionDeclCheck::check(const MatchFinder::MatchResult &Result) {
     if (MethodDecl->getParent()->isLambda())
       return;
 
+  // Ignore implicit functions (e.g. implicit constructors or destructors)
+  if (FuncDecl->isImplicit())
```

Comment:
Instead using isImplicit better just add:
```
 std::optional<TraversalKind> getCheckTraversalKind() const override {
      return TK_IgnoreUnlessSpelledInSource;
  }
```
to a header file of this check.

Reply:
that works, thank you!